adapters/handlers/rest/configure_api.go

```diff
@@ -377,42 +377,104 @@ func MakeAppState(ctx context.Context, options *swag.CommandLineOptionsGroup) *s
 		migrator.RecountProperties(ctx)
 	}
 
-	// TODO_RAFT START
+	// TODO-RAFT START
 	appState.MetaStore = &fsm
 
-	boostrapper := false
+	// If bootstrap expect is bigger than 0, we are bootstraping a new cluster
+	bootstrapper := appState.ServerConfig.Config.Raft.BootstrapExpect > 0
```

Comment:
The bootstrap-expect flag is specific to the voter in RAFT, and NodeCounter() may not necessarily count the number of voters.

Reply:
Then how do you suggest a node queries other nodes (which address, which port ?) to know how many are online and waiting for bootstrap ? What we give as parameter is a node name and a port (raft port).

The only other way I see is to iterate of the raft join list and check for node presence in the memberlist. Is that what you have in mind ?

Reply:
Synced online, code updated. We want to check in the memberlist for each node in the join list to ensure they are online and then use them to bootstrap the cluster.